.github/workflows/cicd.yaml

```
@@ -119,7 +119,8 @@ jobs:
     if: >-
       github.ref == 'refs/heads/main' ||
       startsWith(github.ref, 'refs/tags/v') ||
-      endsWith(github.ref, '-fulltest')
+      endsWith(github.ref, '-fulltest') ||
+      endsWith(github.ref, '-from-tests-temp')
```

```suggestion
      endsWith(github.ref, '-fulltest')
```
TODO commit this after comprehensive tests pass